### DISCUSSION TITLE
Error on Ubuntu

### DISCUSSION THREAD

**Discussion by johannes-bittner** [simulot/immich-go#485] [General]
I followed the readme instructions, but when I launch the script from cli: 
`sh ./immich-go -server=http://xxx:2283 -key=xxx upload -dry-run mytakeout.zip `
I get the following error: 
`./immich-go: 1: Syntax error: "(" unexpected`
 
Initially, I thought that this would be because of files in my takeout that contain a "(", but even after removing/renaming them, the errro persists ... any ideas?

**Comment by simulot** [comment]
Try without the sh:

```
./immich-go -server=http://xxx:2283 -key=xxx upload -dry-run mytakeout.zip 
```

**Reply by johannes-bittner** [reply]
Works like a charm. Merci beaucoup!

**Comment by bttrfngrs** [comment]
Kindly guide me which version of immich-go do i use on my ubuntu VM.. there are quite a few different linux builds.. thanks in advance
